### ISSUE TITLE
provider not available for openbsd_amd64

### ISSUE THREAD

**Issue by eradman** [digitalocean/terraform-provider-digitalocean#533]
After upgrading from terraform 0.12.24 to 0.13.5 the digitialocean provider no longer appears to be available on OpenBSD

```
$ terraform init

Initializing the backend...

Initializing provider plugins...
- Finding latest version of digitalocean/digitalocean...

Error: Failed to install provider

Error while installing digitalocean/digitalocean v2.2.0: provider
registry.terraform.io/digitalocean/digitalocean 2.2.0 is not available for
openbsd_amd64
```

```
$ terraform -v
Terraform v0.13.5
```

**Comment by andrewsomething** [comment]
We've just released v2.3.0 with OpenBSD binaries: https://github.com/digitalocean/terraform-provider-digitalocean/releases/tag/v2.3.0

These should appear in the Terraform registry shortly.

**Comment by eradman** [comment]
Just tested on OpenBSD. Worked perfectly. Thanks!
